can-bus/json/Debug/Debug_alerts.json

```
@@ -0,0 +1,4 @@
+{
+    "msg_id": 700,
+    "alerts": []
+}
```

nit: missing blank line
```suggestion
}

```